tests/go/GoVetBearTest.py

```diff
@@ -20,7 +21,39 @@ def setUp(self):
         self.bad_file = os.path.join(os.path.dirname(__file__),
                                      'test_files',
                                      'vet_bad.go')
+        self.bad_file_two = os.path.join(os.path.dirname(__file__),
+                                         'test_files',
+                                         'vet_bad_two.go')
 
     def test_run(self):
         self.check_validity(self.uut, [], self.good_file)
         self.check_invalidity(self.uut, [], self.bad_file)
+
+    def test_args(self):
+        self.section.append(Setting('assembly_declarations', True))
+        self.section.append(Setting('assign', True))
+        self.section.append(Setting('atomic', True))
+        self.section.append(Setting('boolean', True))
+        self.section.append(Setting('build_tags', True))
+        self.section.append(Setting('invalid_cgo', True))
+        self.section.append(Setting('unkeyed_composite_literals', True))
+        self.section.append(Setting('copying_locks', True))
+        self.section.append(Setting('methods', True))
+        self.section.append(Setting('nil_function', True))
+        self.section.append(Setting('printf', True))
+        self.section.append(Setting('range_loops', True))
+        self.section.append(Setting('shadow_variables', True))
+        self.section.append(Setting('shifts', True))
+        self.section.append(Setting('struct_tags', True))
+        self.section.append(Setting('unreachable', True))
+        self.section.append(Setting('unsafe_pointers', True))
+        self.section.append(Setting('unused_result', True))
+        self.check_validity(self.uut, [], self.good_file)
+        self.check_invalidity(self.uut, [], self.bad_file)
```

Comment:
Maybe use `check_results`?

Reply:
I implemented it in a way in which the already written test for GoVetBear was implemented.

Reply:
kk :+1: